app/controllers/api/v2/config_groups_controller.rb

```diff
@@ -23,6 +23,8 @@ def show
       def_param_group :config_group do
         param :config_group, Hash, :required => true, :action_aware => true do
           param :name, String, :required => true
+          param :puppetclass_ids, Array, :desc => N_("IDs of associated Puppet classes")
+          param :puppetclass_names, Array, :desc => N_("Names of associated Puppet classes")
```

Comment:
When I add a param for ids, hammer adds the param for names automatically, so I am removing this line.